release-notes/testing.yml

```diff
@@ -19,10 +19,10 @@ releases:
         url: "https://github.com/coreos/fedora-coreos-tracker/issues/1823"
       - text: "systemd-systemd unmerge + merge results in broken system"
         url: "https://github.com/coreos/fedora-coreos-tracker/issues/1744"
+  41.20250331.2.0:
+    issues:
```

Comment:
According to https://github.com/coreos/fedora-coreos-streams/commit/79b332b011feeb7d9cb9c718136c49c3cbdf43df this wasn't added to the stream metadata until `42.20250410.2.0`.